### PR TITLE
test(Foldable): test for Foldable added

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/components/Foldable/__tests__/Foldable.test.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import {testCustomClassName} from '../../../../test-utils/shared/common';
+import {getQaAttrubutes} from '../../../utils';
+import Foldable, {FoldableProps} from '../Foldable';
+
+const foldableProps: FoldableProps = {
+    isOpened: false,
+    qa: 'foldable-component',
+};
+
+const IS_OPENED_CLASS_NAME = 'pc-foldable-block_open';
```
Comment (yuberdysheva) [inline src/components/Foldable/__tests__/Foldable.test.tsx:14]:
```suggestion
const IS_OPEN_CLASS_NAME = 'pc-foldable-block_open';
```

File: src/components/Foldable/__tests__/Foldable.test.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+const qaAttributes = getQaAttrubutes(foldableProps.qa);
+
+const FoldableComponent = (props: FoldableProps) => {
+    return (
```
Comment (yuberdysheva) [inline src/components/Foldable/__tests__/Foldable.test.tsx:19]: maybe without return?

File: src/components/Foldable/__tests__/Foldable.test.tsx (hunk 1 above)
Comment (niktverd) [reply src/components/Foldable/__tests__/Foldable.test.tsx:14]: ok

File: src/components/Foldable/__tests__/Foldable.test.tsx (hunk 2 above)
Comment (niktverd) [reply src/components/Foldable/__tests__/Foldable.test.tsx:19]: ok

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**